src/app/clusters/window-covering-server/window-covering-server.cpp

```diff
@@ -728,6 +694,24 @@ bool emberAfWindowCoveringClusterDownOrCloseCallback(app::CommandHandler * comma
         Attributes::TargetPositionTiltPercent100ths::Set(endpoint, WC_PERCENT100THS_MAX_CLOSED);
     }
     emberAfSendImmediateDefaultResponse(EMBER_ZCL_STATUS_SUCCESS);
+
+    Delegate * delegate = GetDelegate(endpoint);
+    if (delegate)
+    {
+        if (HasFeature(endpoint, Feature::kPositionAwareLift))
+        {
+            LogErrorOnFailure(delegate->HandleMovement(WindowCoveringType::Lift));
+        }
+        else if (HasFeature(endpoint, Feature::kPositionAwareTilt))
```

Comment:
This is incorrect we can have both at the same time

Reply:
Yes, we could have both, but the spec say when we adjust the position for lift we should also adjust the position for tilt accordingly, same to the other way,  so if both of them are supported, we only need to change one of them

Reply:
Ack, UpOrOpen/DownOrClose/StopMotion commands will impact both actuator at the same time, and other commands will only adjust one of them